apparmor.d/profiles-a-f/blueman

```diff
@@ -61,6 +62,7 @@ profile blueman @{exec_path} flags=(attach_disconnected) {
   /dev/shm/ r,
   /dev/tty rw,
 
+  deny @{lib}/python3/dist-packages/blueman/__pycache__/** w,
```

Comment:
Don't deny this. It is usually required when python need to recompile bytecode.

Also, use `@{python_name}` instead of `python3` as some distributions have `python3.x` path structure.